core/server/master/src/main/java/alluxio/master/file/meta/UfsAbsentPathCache.java

```diff
@@ -41,4 +46,23 @@
    * @return true if the path is absent according to the cache
    */
   boolean isAbsent(AlluxioURI path);
+
+  /**
+   * Factory class for {@link UfsAbsentPathCache}.
+   */
+  final class Factory {
+    private static final Logger LOG = LoggerFactory.getLogger(UfsAbsentPathCache.Factory.class);
+
+    private Factory() {} // prevent instantiation
+
+    public static UfsAbsentPathCache create(MountTable mountTable) {
+      int numThreads = Configuration.getInt(PropertyKey.MASTER_UFS_PATH_CACHE_THREADS);
+      if (numThreads <= 0) {
+        LOG.warn("UfsAbsentPathCache is disabled. " + PropertyKey.MASTER_UFS_PATH_CACHE_THREADS
+            + " is invalid: " + numThreads);
```

Comment:
`LOG.info` here and change the wording since disabling the cache is a valid configuration?